src/validation.cpp

```diff
@@ -3196,6 +3196,22 @@ std::vector<unsigned char> GenerateCoinbaseCommitment(CBlock& block, const CBloc
     return commitment;
 }
 
+//! Returns last CBlockIndex* that is a checkpoint
+static CBlockIndex* GetLastCheckpoint(const CCheckpointData& data)
+{
+    const MapCheckpoints& checkpoints = data.mapCheckpoints;
+
+    for (const MapCheckpoints::value_type& i : reverse_iterate(checkpoints))
```

Comment:
Might be a nitpick (sorry, new to reviewing here), but single-char var names are not great. Why not use `checkpoint` here instead of `i`. Getting to more readable source will get more devs involved. Doesn't make a huge difference in this tiny loop but it sets an example for others.

Reply:
@skwp this code was moved from `src/checkpoints.cpp` and we try to not change moved code.

Reply:
Thanks @promag, exactly this!

Reply:
would it be ok to create var renaming PRs if keeping code in place?